astroid/brain/brain_pathlib.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2022 Deepyaman Datta <deepyaman.datta@utexas.edu>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+
+from typing import Iterator, Optional
+
+from astroid import bases, context, inference_tip
+from astroid.const import PY310_PLUS
+from astroid.exceptions import UseInferenceDefault
+from astroid.manager import AstroidManager
+from astroid.nodes.node_classes import Attribute, Slice, Subscript
+from astroid.nodes.node_ng import NodeNG
+
+
+def _looks_like_parents_subscript(node: NodeNG) -> bool:
```

Comment:
It has been quite some time that I looked into this myself, but I believe `register_transform` makes it so that only the class passed as first argument is pass to this function.

Perhaps you could do this and run pytest:
```python
if not isinstance(node, nodes.Subscript):
    breakpoint()
```

If there is no break then we can update this to be `nodes.Subscript` instead.

Reply:
Got it, tested it and (of course) it's as you all said!